bindings/python/iota_sdk/client/_utils.py

```diff
@@ -1,20 +1,26 @@
 # Copyright 2023 IOTA Stiftung
 # SPDX-License-Identifier: Apache-2.0
 
+from typing import Optional
+from abc import ABCMeta, abstractmethod
+
 from iota_sdk.types.common import HexStr
 from iota_sdk.types.output import Output
-from typing import Optional
 
 
-class ClientUtils():
+class ClientUtils(metaclass=ABCMeta):
     """Client utility functions.
     """
 
-    def hex_to_bech32(self, hex: HexStr, bech32_hrp: str) -> str:
+    @abstractmethod
+    def _call_method(self, name, data=None):
```

Comment:
Could we also just put some ignore thing here instead? If not please add some comment why it's needed, because this looks useless if one doesn't know for what it is needed

Reply:
we have added the same to 2.0 branch! ill add some docs to all of em then